docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Installing Knative components using Operators"
+weight: 10
+type: "docs"
+---
+
+Knative provides operators as tools to install, configure and manage Knative. This guide explains how to install and
+uninstall Knative using Knative operators.
+
+Each component in Knative has a separate operator for installation and configuration. This means that there is a [Serving operator](https://github.com/knative/serving-operator)
+and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+
+## Before you begin
+
+Knative installation using Operators requires the following:
+
+- A Kubernetes cluster v1.15 or newer, as well as a compatible kubectl. This guide assumes that you've already created
+a Kubernetes cluster. If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB,
+Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least
+4.0 GB, Disk storage to at least 20 GB for each node.
+- The Kubernetes cluster must be able to access the internet, since Knative operators download images online.
+- Istio:
+    - [Download and install Istio](https://knative.dev/development/install/installing-istio/#downloading-istio-and-installing-crds). Go through all the 4 sub-steps.
+    - [Update your Istio to use cluster local gateway](https://knative.dev/development/install/installing-istio/#updating-your-install-to-use-cluster-local-gateway).
+
+## Limitations of Knative Operators:
+
+Knative Operators use custom resources (CRs) to configure your Knative deployment.
+
+ - Currently, the CRs included with Knative Operators do not provide high availability (HA) capabilities.
+ - Knative Operators have not been tested in a production environment, and should be used for development or test purposes only.
+
+## Install Knative Serving with Operator:
+
+Information about Knative Serving Operator releases can be found on the [Releases page](https://github.com/knative/serving-operator/releases).
+
+### Installing the Knative Serving Operator
+
+__From releases__:
+
+Run the following command to install Knative Serving Operator:
+
+```
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/v0.13.0/serving-operator.yaml
```

Comment:
This can be `{{< version >}}` to auto-substitute in the release version, IIRC.

Even better, there is an `{{< artifact >}}` tag, like so:

```suggestion
kubectl apply --filename {{< artifact repo="serving-operator" file="serving-operator.yaml" >}}
```

I'll send this as a PR on the upstream docs, so this is mostly FYI.